apps/app/src/utils/pageScroll.ts

```diff
@@ -1,13 +1,11 @@
 import "client-only";
 
-const classes = ["overflow-hidden"];
+const classes = ["overflow-hidden", "sm:overflow-auto"];
```

Comment:
Teraz zauważyłem, że dodanie `sm:overflow-auto` rozwiązuje problem zablokowanego scrolla, gdy na telefonie przełączamy widok z wertykalnego na horyzontalny. Jednak rozwiązanie to powoduje, że gdy otwieramy modal np. 'Dodaj pytanie' fuckcja `lockScroll` nie oddziałuje na stronę - widoczne są dwa scroll bary, jeden do modala, a drógi do strony.  Proponuję zatem, aby do funkcji `lockScroll` przekazywać parametr który określać, czy scroll bar ma być permanentnie wyłączony (możliwość przewijania strony), czy dla jakiegoś przypadku np. dla szerokich wyświetlaczy ma zostać odblokowany / ma się pojawić. Problem w mojej opinii polega na tym, że chcemy uzyskać reużywalne rozwiązanie do blokowania przewijania strony, ale ciężko to uzyskać, aby było responsywne.